jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Callback.java

```diff
@@ -155,23 +157,33 @@ static Callback from(InvocationType invocationType, Runnable success, Consumer<T
     {
         return new Callback()
         {
+            private final AtomicBoolean _completed = new AtomicBoolean();
```

Comment:
We have `CountingCallback` that can be used with `count=1` with the same behavior, so I would use that and revert these changes to `Callback`.